src/lib/consensus/proof_of_stake.ml

```diff
@@ -177,12 +178,13 @@ module Make (Inputs : Inputs_intf) :
           ( Inputs.probable_slots_per_transition_count
           * Inputs.unforkable_transition_count )
 
+      let after_lock_checkpoint (slot : t) =
+        let open UInt32.Infix in
+        slot < unforkable_count * UInt32.of_int 2
```

Comment:
wouldn't the slot after the lock checkpoint if its greater than F*2?

Reply:
Oh, yeah. That's a big miss. Thanks. Think I swapped out the comparison operator and forgot to switch the arguments.